lesson_13/bank/bank_app/src/main/java/com/codedifferently/lesson13/bank/BusinessCheckingAccount.java

```diff
@@ -0,0 +1,71 @@
+package com.codedifferently.lesson13.bank;
+
+import java.util.Set;
+
+import javax.naming.spi.DirStateFactory;
+
+import com.codedifferently.lesson13.bank.exceptions.NoBusinessOwnersException;
+import com.codedifferently.lesson13.bank.exceptions.InsufficientFundsException;
+
+public class BusinessCheckingAccount extends CheckingAccount {
+    
+    public BusinessCheckingAccount(String accountNumber, Set<Customer> owners, double initialBalance) {
+    super(accountNumber, owners, initialBalance);
+    if (checkOwners(owners) == 0) {
+      throw new NoBusinessOwnersException("A business owner must be one of the customers opening a business account.");
+    }
+    this.accountNumber = accountNumber;
+    this.owners = owners;
+    this.balance = initialBalance;
+        isActive = true;}
+    private int checkOwners(Set<Customer> owners) {
```

Comment:
Pretty good attempt, nice. You gotta fix your formatting though, your code is hard to read when you're not following the rules.